pom.xml

```diff
@@ -145,7 +145,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
                 <configuration>
-                    <tagNameFormat>pipeline-stage-view-@{project.version}</tagNameFormat>
+                    <tagNameFormat>pipeline-graph-analysis-@{project.version}</tagNameFormat>
```

Comment:
:ant: Delete this whole section. You probably copy-pasted it from `pipeline-stage-view-plugin`, which was a multimodule repository.